.circleci/config.yml

```
@@ -39,7 +39,7 @@ jobs:
           command: helm dependency update
       - run:
           name: Helm install stackstorm-ha chart
-          command: helm install --timeout 10m0s --debug --wait --name-template stackstorm-ha .
+          command: helm install --timeout 20m0s --debug --wait --name-template stackstorm-ha .
```

I didn't see any evidence that lifting install timeout helps.
Usually from the history of CI failures if deployment didn't complete in ~5mins, everything goes into a crash loop which is rarely recoverable without a manual intervention.
The usual reason behind the crash loop is one of the backend clusters didn't startup correctly like MongoDB, Redis or RabbitMQ.

But let's see if this helps.
Please use at least `15m` everywhere as bumping to `20mins` is too much!


```suggestion
          command: helm install --timeout 15m0s --debug --wait --name-template stackstorm-ha .
```